clouddriver-core/src/test/java/com/netflix/spinnaker/clouddriver/cache/CatsSearchProviderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Salesforce.com, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.cache;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.netflix.spectator.api.NoopRegistry;
+import com.netflix.spectator.api.Registry;
+import com.netflix.spinnaker.cats.cache.Cache;
+import com.netflix.spinnaker.cats.mem.InMemoryCache;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.context.annotation.UserConfigurations;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.context.annotation.Bean;
+
+public class CatsSearchProviderTest {
+  private final ApplicationContextRunner runner =
+      new ApplicationContextRunner()
+          .withPropertyValues("caching.write-enabled=false", "redis.enabled:false")
+          .withConfiguration(UserConfigurations.of(CacheConfig.class, TestConfiguration.class))
+          .withAllowBeanDefinitionOverriding(true);
```

Comment:
Note to self...try to remove this as part of https://github.com/spinnaker/clouddriver/pull/5705